source/features/show-names.tsx

```diff
@@ -84,7 +84,6 @@ void features.add(import.meta.url, {
 	include: [
 		pageDetect.hasComments,
 	],
-	onlyAdditionalListeners: true,
```

Comment:
The feature was inadvertently disabled by #5873 (https://github.com/refined-github/refined-github/pull/5873/files#diff-bb287cee7084ff807d23e0ac0cdee19825f4c215ce976f3ab5f608448c4bc8eaR87)